lib/start.js

```diff
@@ -16,7 +16,7 @@
 
 'use strict'
 
-process.env.DEBUG = `sdk:usbboot,${process.env.DEBUG}`
+process.env.DEBUG = `sdk:usbboot${process.env.DEBUG ? `,${process.env.DEBUG}` : ''}`
```

Comment:
Can you test if something like this works?

```js
process.env.DEBUG = `sdk:usbboot${process.env.DEBUG || ''}`
```

Reply:
Ah, but that leaves the trailing comma (which doesn't really matter, but its a bit ugly). Up to you, I'll approve it anyways :)

Reply:
The missing comma would complicate things wouldn't it? I was thinking

```js
[' sdk:usbboot', process.env.DEBUG ].filter(_ => _).join(',')
```
but the linter would complain about non-lodash usage.

Reply:
I guess the 'proper' way to do it would be to split `process.env.DEBUG` by `,` into an array, append `sdk:usbboot` to that array, and then join the array with `,` back into a string. But that's obviously a lot of work when @Shou 's simple inline-conditional works fine :)